_episodes/03-matplotlib.md

```diff
@@ -13,15 +13,28 @@ keypoints:
 ---
 
 ## Visualizing data
-The mathematician Richard Hamming once said, "The purpose of computing is insight, not numbers," and
+The mathematician Richard Hamming once said, "The purpose of computing is insight, not numbers", and
 the best way to develop insight is often to visualize data.  Visualization deserves an entire
 lecture of its own, but we can explore a few features of Python's `matplotlib` library here.  While
 there is no official plotting library, `matplotlib` is the _de facto_ standard.  First, we will
-import the `pyplot` module from `matplotlib` and use two of its functions to create and display a
+import the `pyplot` module from `matplotlib`, and then we will use two of its functions to create and display a
 [heat map]({{ page.root }}/reference.html#heat-map) of our data:
 
+> ## Important Note
+>
+> In the previous lesson we set the `data` variable and imported `numpy`. If you have not completed lesson `2. Analyzing Patient Data`,
+> please run the following commands prior to continuing with the current lesson.
+>
+> ~~~
+> import numpy
+> data = numpy.loadtxt(fname='inflammation-01.csv', delimiter=',')
+> ~~~
+> {: .language-python}
+{: .callout}
```

Comment:
`.callout` is fine but I wonder if it should be a `.prereq`. What do you think, @ldko?

Reply:
I didn't realize `.prereq` was a thing until you mentioned it, and I looked for it :). I think it makes sense to use that for consistency with what is given as `Prerequisites` at https://swcarpentry.github.io/python-novice-inflammation/.